Vulkan/glslang-build/glslang-build.vcxproj

```diff
@@ -14,9 +14,9 @@
     <ProjectGuid>{8F85B6CC-250F-4ACA-A617-E820A74E3E3C}</ProjectGuid>
     <Keyword>MakeFileProj</Keyword>
   </PropertyGroup>
-  <Import Project="..\..\common_default.props" />
+  <Import Project="$(SolutionDir)\buildfiles\msvc\\common_default.props" />
```

Comment:
Here you have double backslash, but in every other place, it a single backslash
```suggestion
  <Import Project="$(SolutionDir)\buildfiles\msvc\common_default.props" />
```

And also in the changeset below.

Reply:
wouldn't it make more sense to put the glslang props file into the same folder as project.
a.k.a. Vulkan/glslang_build

Reply:
"wouldn't it make more sense to put the glslang props file into the same folder as project.
a.k.a. Vulkan/glslang_build"

it does, but that'll be done in a separate commit sometime :)

Reply:
why

Reply:
because buildsystem changes can break the build if not properly reviewed; keeping pull requests makes code reviews easier

Reply:
which is why i don't want to review this a second time

Reply:
the gslang-build file wasnt moved in this commit

Reply:
I think it was:
![image](https://user-images.githubusercontent.com/23019877/117486889-8eac9000-af6a-11eb-8ad4-fe7b24e7a07d.png)

Reply:
im confused :)

Reply:
Maybe it makes more sense if you read the first comment again xD
"wouldn't it make more sense to put the glslang props file into the same folder as project.
a.k.a. Vulkan/glslang_build"

Reply:
ohhhhhh 

i thought you meant the project file!! im dumb 

that depends, i just thought id lump them together for now. its down to what yall prefer

Reply:
usually if you create a property sheet in VS, the default folder is the same as the location of that project's vsxproj file (or what it is called)

Reply:
okay, ill move the gslang stuff around